<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1557</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowkind" stdset="0">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="digests_table">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>61</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Digest</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="owner_ids_table">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>141</width>
      <height>430</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>owner_ids</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="agendas_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>171</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>econ Stories</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="depotlinks_table">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>191</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Agendas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weight</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="econ_delete_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete </string>
    </property>
   </widget>
   <widget class="QPushButton" name="econ_insert_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new econ</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="econ_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="econ_update_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change PID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agenda_update_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change Agenda PID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agenda_insert_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new Agenda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agenda_delete_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="agenda_owner_id">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="owner_id_update_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change owner_id PID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="owner_id_insert_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new owner_id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="owner_id_delete_button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete owner_id</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="person_id">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_delete_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>delete Agenda</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depotlink_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_insert_button">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add new Agenda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_update_button">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update Agenda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_all_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="agenda_edit_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit </string>
    </property>
   </widget>
   <widget class="QPushButton" name="owner_id_edit_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit owner_id</string>
    </property>
   </widget>
   <widget class="QPushButton" name="depotlink_edit_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit Agenda</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="w_intent_table">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>260</y>
      <width>371</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>owner_ids Agenda</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_partys_table">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>owner_ids Partys</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_groups_table">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>260</y>
      <width>171</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>owner_ids Groups</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_ideas_table">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>391</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>owner_id Ideas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Idea parent_road</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="w_beliefs_table">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>371</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Beliefs Base</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Beliefs Belief</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QComboBox" name="econ_name_combo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="econ_copy_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Copy</string>
    </property>
   </widget>
   <widget class="QComboBox" name="depotlink_type_combo">
    <property name="geometry">
     <rect>
      <x>376</x>
      <y>110</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="econ_load_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="depotlink_weight">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Agenda Weight:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ignores_table">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>61</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Ignore</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="show_digests_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>digest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_ignores_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ignore</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_ignore_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_ignore_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>160</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_job_agenda_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>output_agenda to jobs_dir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reload_jobs_job_agendas_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>620</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>refresh_job_agendas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_jobs_and_reload_srcs_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>dest2jobs and refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="role_open_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>role</string>
    </property>
   </widget>
   <widget class="QPushButton" name="role_save_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1557</width>
     <height>26</height>
    </rect>
   </property>
   <property name="accessibleDescription">
    <string>General Operations</string>
   </property>
   <property name="accessiblePID" stdset="0">
    <string>File</string>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string/>
    </property>
    <property name="kind" stdset="0">
     <string>File</string>
    </property>
    <addaction name="fm_new"/>
    <addaction name="fm_open"/>
    <addaction name="fm_save"/>
    <addaction name="save_as"/>
   </widget>
   <addaction name="file_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fm_open">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="fm_save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="save_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="fm_new">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
